src/swarm/Swarm.ts

```diff
@@ -78,19 +78,29 @@ const imageMimeTypeFromPath = (path: string): string => {
 };
 
 const uploadPhoto = async (localPath: string, swarmGateway: string = defaultGateway): Promise<string> => {
-    Debug.log('uploadPhoto: ', localPath);
+    return uploadPhotos([{localPath, name: 'photo'}], swarmGateway);
+};
+
+interface Photo {
+    localPath: string;
+    name: string;
+}
+
+const uploadPhotos = async (photos: Photo[], swarmGateway: string): Promise<string> => {
```

Comment:
I missed this earlier but calling these things Image/Picture is more descriptive, Photo is unnecessarily exclusive.

Reply:
I renamed it to uploadFiles, because that's what it actually does.

https://github.com/agazso/postmodern/pull/178/commits/fdb1394ca980e83cbe31fcfc6f4ca153991bd767